.github/workflows/delete-namespace.yml

```diff
@@ -63,12 +63,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: checkout repo
-      uses: actions/checkout@v2
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11  #v4.1.1
```

Comment:
Have you tested everywhere that this newer major version works for us like before?

Reply:
I tested the new commit hash one the deployment of  the spsh-client Helm chart. The CI/CD pipeline ran successfully. You can check this workflow. By the way, trivy scan failed, but the cause is not the commit hash; rather, it's due to CVE-2023-52425.
https://github.com/dBildungsplattform/schulportal-client/actions/runs/7895165196

Reply:
this has not been tested here: https://github.com/dBildungsplattform/spsh-app-deploy/actions/workflows/delete-namespace-manual.yml

Reply:
https://github.com/dBildungsplattform/spsh-app-deploy/actions/runs/7914749618